### DISCUSSION TITLE
How to access older cases then your account

### DISCUSSION THREAD

**Discussion by Poul-B** [globaleaks/globaleaks-whistleblowing-software#4127] [Q&A] [answered]
Hey, I'm new to this forum, but I hope that you can help.

I have a case in which a newly created user, doesn't have access to a case. The case is grayed-out, so the user can’t open it. 
From what I have been told, a user doesn’t have access to cases created before this user’s user account were created and to get access, that user needs another user which have access, to give access to the case. 

But what if there aren’t another user?
What if the other user account was deleted before the new user got her account, so she is the only user on the site?

**Comment by evilaliv3** [comment, marked as answer]
Hello! Thank you for your question.

Reports are delivered to users that are configured on the system when the report is received and those are the only ones with access to those reports and the only ones that can eventually transfer access to new users.

In case of a system with only one recipient, if this user is deleted the data is deleted with them.

**Reply by Poul-B** [reply]
Can you please clarify …the data is deleted with them?

What happens if an older recipient, who have access to the report, is deleted, while there are other recipients active but which had been created after the report were generated?
Then there are no recipients which have access to the report and no recipients left to give that access.

Is the report then deleted with the deletion of the old recipient?

**Reply by evilaliv3** [reply]
@Poul-B as explained reports are delivered to recipients existing at the time the report is received and they are the only one that can access that report and grant access to others. If you delete them before they grant access to others the report is deleted.

So yes, if you deleted a user before they transfered the access to other user, the reports that that solely user had access to have been deleted.

**Reply by Poul-B** [reply]
@evilaliv3 Thank you for the additional explanation.

This is at a very important aspect of the system, though I understand the security argument, the fact that there is a risk of losing a report if recipient management isn’t careful, is something that need to be addressed internally. 

And that is something I can do :)
